applications/serial_lte_modem/Kconfig

```diff
@@ -27,6 +27,9 @@ config SLM_AT_MAX_PARAM
 config SLM_NATIVE_TLS
 	bool "Use Zephyr mbedTLS"
```

Comment:
Actually, the right spelling is Mbed TLS.

Reply:
Correct, but should not be part of this PR as we have not made any changes in this part. Thx.